lib/logger/logger.go

```diff
@@ -62,11 +93,38 @@ func setLoggerOutput() {
 		output = os.Stderr
 	case "stdout":
 		output = os.Stdout
+	case "syslog":
+		output = setSyslogConnection()
 	default:
 		panic(fmt.Errorf("FATAL: unsupported `loggerOutput` value: %q; supported values are: stderr, stdout", *loggerOutput))
 	}
 }
 
+func setSyslogConnection() *syslog.Writer {
```

Comment:
I would suggest moving this funciton into `lib/syslog` and name it `getSyslogWriter`.
This will help to decouple logger from the syslog implementation